ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,126 @@
     "FieldType": "Document",
     "ElementLabel": "Upload a file",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "emailAddress",
+    "ListElementCode": "email",
+    "FieldType": "Text",
+    "ElementLabel": "Email Address",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "telephone",
+    "ListElementCode": "telephoneNumber",
+    "FieldType": "Text",
+    "ElementLabel": "Telephone",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "telephone",
+    "ListElementCode": "contactDirection",
+    "FieldType": "Text",
+    "ElementLabel": "Person to contact",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "mobileNumber",
+    "ListElementCode": "telephoneNumber",
+    "FieldType": "Text",
+    "ElementLabel": "Mobile Number",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ApplicantParty",
+    "ListElementCode": "partyId",
+    "FieldType": "Text",
+    "ElementLabel": "Party ID",
+    "FieldShowCondition": "partyType=\"DO_NOT_SHOW\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ApplicantParty",
+    "ListElementCode": "partyType",
+    "FieldType": "Text",
+    "ElementLabel": "Party type",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ApplicantParty",
+    "ListElementCode": "name",
```

Comment:
What happened to `firstName` / `lastName`? Spec says we should not capture full name.

Reply:
in the applicant spec name seems to be name of the authority and not a persons name. See attached
![applicantDataChange](https://user-images.githubusercontent.com/42037338/60725360-cc849b80-9f30-11e9-94a9-b5697bcba858.png)

Reply:
@swalker125

Reply:
So as far as i can see first name and last name would not make sense here. In the other structures first name and last name is specified in this one it is one name and so it would not make sense to split it up. @geek-soft @swalker125

Reply:
there is an organisationName in party structure. Our applicants are always Local Authorities

Reply:
agree first and last name does not make sense in this scenario.

Reply:
ok organisationName seems to be best fit @swalker125

Reply:
👍